File: Lista2/Java/Romb.java
public class Romb extends Czworokat
{
    private double bok;
    private double kat;

    Romb(double bok, double kat)
    {
        this.bok = bok;
        this.kat = kat;
    }

    public double obwod()
    {
        return (4 * bok);
    }

    public double pole()
    {
        return (Math.pow(bok, 2) * Math.sin(Math.toRadians(kat)));
    }
}

File: Lista2/C++/Romb.cpp
#include <iostream>
#include <cmath>
#include "Romb.h"

Romb::Romb(double bok, double kat)
{
    this->bok = bok;
    this->kat = kat;
}

double Romb::obwod()
{
    return (4 * bok);
}

double Romb::pole()
{
    return (pow(bok, 2) * sin(kat * (3.14/180)));
}

Romb::~Romb()
{
    
}
